python/ql/src/experimental/Security/CWE-295/UrlopenWithoutValidation.ql

```diff
@@ -0,0 +1,45 @@
+/**
+ * @name Attempt to open url without certificate validation
+ * @description Opening a url without certificate validation can allow man-in-the-middle attacks.
+ * @kind problem
+ * @problem.severity error
+ * @id py/urlopen-without-cert-validation
+ * @tags security
+ *       external/cwe/cwe-295
+ */
+
+import python
+import semmle.python.ApiGraphs
+
+predicate calls_urllib(CallNode call) {
+  call =
+    API::moduleImport(["urllib", "urllib2"])
+        .getMember("request")
+        .getMember("urlopen")
+        .getACall()
+        .getNode()
+}
```

Comment:
```suggestion
predicate calls_urllib(DataFlow::CallCfgNode call) {
  call =
    API::moduleImport(["urllib", "urllib2"]).getMember("request").getMember("urlopen").getACall()
}
```
For what you're doing (and in line with my other suggestions) there's no need to go back to the CFG layer for this, so we can just return the `CallCfgNode` we get from `getACall` directly.